umm-spec-lib/src/cmr/umm_spec/test/expected_conversion_util.clj

```diff
@@ -222,3 +223,15 @@
                  (update pr :OnlineResource dif-online-resource)
                  pr))
       check-nil-pub-ref))
+
+(defn expand-file-dist-infos
+  "Parse each format to see if the format contains multiple values. If it does
+   then replace the distribution info that exists and add new ones for each
+   multple format that exists."
+  [file-dist-info]
+  (let [formats (get (select-keys file-dist-info [:Format]) :Format)
```

Comment:
You don't need to do the `select-keys` here. Just do this
`(let [formats (:Format file-dist-info)]`

Reply:
Thanks. I fixed the code.